AnnoyancesFilter/sections/antiadblock.txt

```diff
@@ -382,13 +382,13 @@ usrowing.org,stevensducks.com##.sidearm-adblock
+supersoluce.com,geekzone.co.nz,gigazine.net,fresherslive.com,rhymes.net,zilinak.sk,antoloji.com,f5haber.com,log.com.tr,pinkvilla.com,shwoghk14.blogspot.com,europapress.es,guiainfantil.com,cnbcindonesia.com,wordreference.com,spidersweb.pl,dennikn.sk,eksiup.com,sabervivirtv.com,paradurumu.com,sabah.com.tr,hurriyet.com.tr,dn.pt,proprofs.com,express.co.uk,androidpolice.com,gigazine.net,abbreviations.com,lectortmo.com,cuerpomente.com,iflscience.com,woxikon.de,mavikadin.com,milliyet.com.tr,gizbot.com,lyrics.com,dicionario.priberam.org,news.goo.ne.jp,haberler.com,sondakika.com,regnradar.se,agazeta.com.br,thehill.com,quotes.net,cnnindonesia.com,newsbomb.gr,interpals.net,boldsky.com,all3dp.com,detik.com,tecnoblog.net,wenxuecity.com,abola.pt,drivespark.com,mykhel.com,careerindia.com,goodreturns.in,oneindia.com,greensboro.com,ynetnews.com,abc.es,abcnyheter.no,accuweather.com,avsforum.com,bellinghamherald.com,blic.rs,blitz.bg,bloomberght.com,bnd.com,brantfordexpositor.ca,business.financialpost.com,canoe.com,cinemablend.com,cnnturk.com,contra.gr,digitaltrends.com,dumpert.nl,dziennikbaltycki.pl,elmundo.es,elnuevoherald.com,esquire.com.gr,expressilustrowany.pl,fredericksburg.com,fresnobee.com,haberturk.com,hnonline.sk,ign.com,index.hr,interia.pl,izismile.com,jetphotos.com,kansascity.com,kentucky.com,klix.ba,kupujemprodajem.com,ladylike.gr,latimes.com,lecturas.com,lifestyle.bg,listenonrepeat.com,maisfutebol.iol.pt,marca.com,marketscreener.com,metropoles.com,money.bg,montrealgazette.com,mtlblog.com,mynet.com,narcity.com,nationalpost.com,news.bg,news247.gr,ni.dk,ntv.com.tr,nwitimes.com,observador.pt,okdiario.com,omaha.com,phonearena.com,polskatimes.pl,poststar.com,sanluisobispo.com,showtv.com.tr,slickdeals.net,sme.sk,snopes.com,sport24.gr,star-telegram.com,startv.com.tr,sunherald.com,tdn.com,theindependent.com,theobserver.ca,theprovince.com,thesouthern.com,thestate.com,tomshw.it,topito.com,topsport.bg,torontosun.com,trib.com,uol.com.br,vancouversun.com,wcfcourier.com,windsorstar.com,ynet.co.il,yourdictionary.com,zonebourse.com#%#//scriptlet("abort-current-inline-script", "atob", "googlefc")
```

Comment:
Did you check it? It doesn't work on my end, so remove the domain.
Also please add `@@||supersoluce.com^$generichide` a bit lower to the
```
!#if (adguard_app_ios || adguard_ext_android_cb)
!#safari_cb_affinity(general)
@@||fresherslive.com^$generichide
@@||universoformulas.com^$generichide
@@||sneakernews.com^$generichide
@@||sonyliv.com^$generichide
@@||synology-forum.de^$generichide
@@||news.goo.ne.jp^$generichide
@@||agazeta.com.br^$generichide
!#safari_cb_affinity
!#endif
```

Reply:
It works for me

Reply:
What works for you I will add it.

Reply:
![image](https://user-images.githubusercontent.com/76908410/107000626-257a7c80-67c3-11eb-9bfe-99037f4e0c35.png)
After killing the dialog there is another notice at the below of the page just like `geekzone.co.nz` issue.

Reply:
Those CSS rules are enough for the website.

Reply:
@zloyden So should I remove the Scriptlet rule?

Reply:
Yes, but not the scriptlet rule, ofc, just the domain)

Reply:
Well done:)